roles/openshift_management/tasks/validate.yml

```diff
@@ -2,12 +2,23 @@
 # Validate configuration parameters passed to the openshift_management role
 
 ######################################################################
+# BETA ACKNOWLEDGEMENT
+- name: Ensure BETA software notice has been acknowledged
+  assert:
+    that:
+      - openshift_management_install_beta | default(false) | bool
```

Comment:
we said we'd only do this for OCP, `or openshift_deployment_type != 'openshift-enterprise'`